lib/openshift/kata_config.rb

```diff
@@ -0,0 +1,34 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # represents Metering CRD
+  class KataConfig < ClusterResource
+    RESOURCE = 'kataconfigs'
```

Comment:
@pruan-rht , I planned to review but procrastinated :/
you need to add api name to it.

Reply:
@akostadinov how can I find the api name from the `oc get kataconfig -o yaml` [output](http://pastebin.test.redhat.com/921121)?

Reply:
@pruan-rht , see `selfLink`. In this case it is `kataconfiguration.openshift.io`. So you get:
```
RESOURCE = 'kataconfigs.kataconfiguration.openshift.io'
```